docs/source/usersguide/depletion.rst

```diff
@@ -315,11 +315,11 @@ normalizing reaction rates:
    the cross-sections by the ``source-rate``.
 2. ``fission-q`` normalization, which uses the ``power`` or ``power_density``
    provided by the time integrator to obtain reaction rates by computing a value
-   for the flux based on this power. The general equation for the flux is 
+   for the flux based on this power. The general equation for the flux is
 
    .. math::
 
-      \phi = \frac{P}{V \cdot \sum_i (Q_i \cdot \sigma^f_i \cdot \n_i)}
+      \phi = \frac{P}{V \cdot \sum\limits_i (Q_i \cdot \sigma^f_i \cdot n_i)}
 
    where :math:`\sum_i` is the sum over all nuclides :math:`i`. This equation
```

Comment:
```suggestion
   where :math:`\sum\limits_i` is the sum over all nuclides :math:`i`. This equation
```
Perhaps we should also define our terms? It's obvious to us right now, but to a new user it would be good to have each variable defined.